CONTRIBUTING.md

```diff
@@ -60,9 +60,13 @@ Click the icon at the right end of that box to copy that URL.
     git clone https://github.com/<UserName>/keptn.github.io
     or
     git clone git@github.com:<UserName>/keptn.github.io.git
+    or
+    gh repo clone <UserName>/keptn.github.io
     ```
     Where <*UserName*> is your github username. The keptn.github.io directory is now available in the local directory.
 
+> Make sure you have installed [Github CLI](https://cli.github.com/) to follow the last instructions.
```

Comment:
Good catch that we need to include this!  But it would be better to put this before the procedure.  What about adding a sentence like "Be sure that Git is installed on your system" in the lead-in to this procedure, perhaps includikng a link to https://github.com/git-guides/install-git for instructions?

Reply:
Hi, @StackScribe I have updated you about this one too. Have a look on latest commit.

Reply:
This still seems to be in line 68.  I added another comment suggesting it be moved to right after line 40.